data-structures-stacks/max-value.js

```diff
@@ -0,0 +1,18 @@
+/* exported maxValue */
+
+function maxValue(stack) {
+  let max = '';
+  if (stack.peek() === undefined && max === '') {
```

Comment:
The `&& max === ''` check is redundant, you set max to an empty string on the line before so it will always be an empty string in that `if` statement.